packages/python/plotly/_plotly_utils/tests/validators/test_integer_validator.py

```diff
@@ -32,6 +32,13 @@ def validator_max():
 def validator_aok(request):
     return IntegerValidator("prop", "parent", min=-2, max=10, array_ok=True)
 
+@pytest.fixture
+def validator_extras():
+    return IntegerValidator("prop", "parent", min=-2, max=10, extras=['normal', 'bold'])
+
+@pytest.fixture
+def validator_extras_aok():
+    return IntegerValidator("prop", "parent", min=-2, max=10, array_ok=True, extras=['normal', 'bold', 'italics'])
```

Comment:
Let's drop `italics` from extras in test which could be little confusing. 
This is mainly used for `text-weight` that has only `normal` and `bold` as extras.